workspacemanager/WorkspaceManagerPresenter.py

```diff
@@ -0,0 +1,75 @@
+from workspacemanager.command import Command
+import os.path
+
+
+#TODO askOwen is passing mtd.getObjectNames() ideal? pass mtd? Answer: Will do for time being
```

Comment:
Pass a list of strings to the View for display. As a word of warning. In other operations you will need to pass the actual workspaces, so don't try to solve every problem with strings.